src/main/java/kr/heesu/lotto/domain/Lottos.java

```diff
@@ -22,9 +22,9 @@ public static Lottos of(Set<Lotto> lottos) {
         return new Lottos(lottos);
     }
 
-    public RankResult matches(WinningNumbers winningNumbers, LottoNumber bonusBall) {
+    public RankResult matches(WinningLotto winningLotto) {
         Map<Rank, Long> matches = lottos.stream()
-                .map(lotto -> lotto.match(winningNumbers, bonusBall))
+                .map(lotto -> lotto.match(winningLotto))
```

Comment:
앞서 언급한 것 처럼 WinningLotto에서 처리할 수 있다면
```suggestion
                .map(winningLotto::match)
```
이런식으로 작성할 수 있습니다!